### PR TITLE
#50: Project update to MS v2024.01.02

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR updates the MS submodule version to v2024.01.02 along with
- print scale in `config.yaml`
- version

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Enhancement

## Issue

**What is the current behavior?**
- #50 

**What is the new behavior?**
MS submodule updated to v2024.01.02 and other related migrations in the project

### DB Changes
Follow the migration details with respect to DB/schema changes:

-  Resource metadata to include information about resource creator/editor and **advertised/unadvertised** info

   In detail migration information is available [here](https://docs.mapstore.geosolutionsgroup.com/en/v2024.01.00/developer-guide/mapstore-migration-guide/#database-update) 

    **POSTGRESQL** 
     - https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/postgresql/postgresql-migration-from-v.2.0.0-to-v2.1.0.sql
     
    **ORACLE**
    - https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/oracle/oracle-migration-from-v.2.0.0-to-v2.1.0.sql
    
    **H2**
    - https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/h2/h2-migration-from-v.2.0.0-to-v2.1.0.sql

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: configs/localConfig.json [hunk 1]
```diff
@@ -8,7 +8,19 @@
       "https://nominatim.openstreetmap.org",
       "http://cloudsdi.geo-solutions.it",
       "https://gs-stable.geo-solutions.it/geoserver",
-      "https://gs-stable.geo-solutions.it:443/geoserver"
```
Comment (MV88) [inline configs/localConfig.json:11]: i'm not sure we have to spread these in our clients projects

File: web/pom.xml [hunk 2]
```diff
@@ -15,65 +15,76 @@
...
+    <geostore-webapp.version>2.1.0</geostore-webapp.version>
+    <http_proxy.version>1.5.0</http_proxy.version>
+    <print-lib.version>2.4-SNAPSHOT</print-lib.version>
+    <httpclient.version>4.5.13</httpclient.version>
+    <junit.version>4.13.1</junit.version>
+    <mockito-all.version>1.9.5</mockito-all.version>
+    <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
+    <ehcache-web.version>2.0.4</ehcache-web.version>
+    <commons-pool.version>1.5.4</commons-pool.version>
+    <jaxws-api.version>2.3.1</jaxws-api.version>
```
Comment (MV88) [inline web/pom.xml:28]: reuse these version values in the version below

**Review by MV88 — CHANGES_REQUESTED** [review]

**Review by MV88 — CHANGES_REQUESTED** [review]
please restore use cors entries.

we might keep them

**Review by MV88 — APPROVED** [review]

**Comment by tdipisa** [comment]
@EddyCatt I will wait for your green light before merging this PR at the beginning of September as agreed.

**Pull request merged**